src/components/NewRuntimeModal.js

```diff
@@ -821,7 +821,7 @@ export const NewRuntimeModal = withModalDrawer({ width: 675 })(class NewRuntimeM
           id => h(div, { style: { display: 'flex', flexDirection: 'column' } }, [
             label({ htmlFor: id, style: styles.label }, ['Persistent disk size (GB)']),
             div({ style: { marginTop: '0.5rem' } }, [
-              'Stores your analysis data. ',
+              'stores your analysis data. ',
```

Comment:
Same as above re capitalization

Reply:
Can this be capital `S`?

Reply:
Made it a full sentence to be consistent with Galaxy modal's wording.

![Messages Image(696994228)](https://user-images.githubusercontent.com/5438223/113873844-2dcf4380-9783-11eb-823c-d6bf137233ff.jpeg)